src/services/books/books-service.ts

```diff
@@ -42,7 +42,7 @@ export interface Book {
     name: string;
     email: string;
   };
-  resume: string;
+  resume: string | null;
```

Comment:
Please, cancel it. On the backend is only string type/ I will change it in Trello.

Reply:
Fixed